### ISSUE TITLE
FluidLoggedAPI weird behavior with stairs

### ISSUE THREAD

**Issue by G1ll0u** [Elite-Modding-Team/MoCreaturesExtended#186]
Hello !

I use [FluidLoggedAPI](https://github.com/jbredwards/Fluidlogged-API).
When my fish goes towards a fluid logged stair here's what happens :

![image](https://github.com/user-attachments/assets/cb14d44b-792e-45ac-8352-9690a2d26109)

sometimes my fish runs aground and tries to kill himself... I had to give him mouth-to-gills resuscitation.

Have a nice day.

**Comment by G1ll0u** [comment]
Oh, I use Cleanroom MC with this modlist :
https://mclo.gs/D8c3FEB
Sorry, I forgot that.

**Comment by IcarussOne** [comment]
@G1ll0u
Fluidlogged API has recently had an entire rewrite since v3.0.0. Does this occur in v3.0.4+?

**Comment by G1ll0u** [comment]
Yes, and sometimes fishes kill themselves when reaching stairs by running aground out of the water...

Here's a new video (don't care about non-transparent waterlogged block, this problem is already reported) :

https://github.com/user-attachments/assets/eeae33cf-f102-4d97-93d5-22af37baaf8e

**Comment by IcarussOne** [comment]
This is intentional behavior, the fluidlogged stairs are simply too small for the fish the swim through them. They should properly work on fluidlogged blocks with enough space.
